doc-content/enterprise-only/installation/patches-applying-proc.adoc

```diff
@@ -4,7 +4,7 @@
 
 Automated update tools are often provided with both patch updates and new minor versions of {PRODUCT} to facilitate updating certain components of {PRODUCT}, such as {CENTRAL}, {KIE_SERVER}, and the {HEADLESS_CONTROLLER}. Other {PRODUCT} artifacts, such as the {DECISION_ENGINE} and standalone {CENTRAL}, are released as new artifacts with each minor release and you must re-install them to apply the update.
```

Comment:
Should "re-install" be "reinstall"? Style guide is not specific but it does say "reinstall" throughout the guide.

Reply:
I changed it. The general rule is if it is 'weird' because of vowels next to vowels, use the hyphen..